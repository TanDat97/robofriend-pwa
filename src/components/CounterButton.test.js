import React from 'react';
import { shallow, mount, reder } from 'enzyme';
import CounterButton from './CounterButton';

it('snapshot counterbutton component', () => {
  const mockColor = 'red';
  expect(shallow(<CounterButton color={mockColor} />).debug()).toMatchSnapshot();
})

it('correct increment counter', () => {
  const mockColor = 'red';
  const wrapper = shallow(<CounterButton color={mockColor} />);
  wrapper.find('[id="counter"]').simulate('click');
  expect(wrapper.state()).toEqual({count: 1});
})