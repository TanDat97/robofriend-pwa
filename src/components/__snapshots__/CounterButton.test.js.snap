// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot counterbutton component 1`] = `
"<button id=\\"counter\\" color=\\"red\\" onClick={[Function]}>
  Count: 
  0
</button>"
`;
